### PR TITLE
Milestone 4: Building with Backgrounds and Gradients

### PR DESCRIPTION
**Pull request by hillarioh**
This is a solo project entailing

- creating a clone apple.com site

- Using background styling and gradient

### CODE REVIEW

**Comment by lekegitrepo** [comment]
### Hi @hillarioh 👏 👏 
### You did a great job. The page looks almost like the original but some contents/requirements are missing which I think when implemented will make the page look better.

**_1. Enable stickler for the repo._**

**_2. The first eight elements should be links (to other sections or `#`), and the search bar should be an_** `<input>` tag ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

**_3. The four images at the bottom of the page should be links (to other articles or `#`)_** ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

**_4. Elements should be aligned with_** [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/align_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59538830/))

**_Float and Grid are missing in your style rules._**

**_5. Try to improve your Html semantic by moving footer out of the main and put it below or after main._**

### Optional:
**_1. Try to improve the navbar: let the contents (menu items) have a little bit of space horizontally._**

**_2. Change Author2 in the readme to the Author._**

**_3. Fix the live demo link._**

**_If you have any doubt or questions contact me on slack._**

### Reviewed by: _Temitayo Adeleke._

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
### Status: Require changes.

**Comment by hillarioh** [comment]
I have tried to implement flex on the header ul, and grid on the images in class button, I have edited the readme file and the live link will be available once I merge with the master.

**Comment by hillarioh** [comment]
The link is working now

**Comment by Dandush03** [comment]
# :red_circle: Status: Required Changes :red_circle:
### Hey @hillarioh     :wave: 
### Good attempts on this one :muscle: :+1: 

## Your project looks great! but is a very important thing you have to consider before submitting another code review :point_down: 

- [ ]  :one: You need to [Set-up Linter](https://github.com/microverseinc/linters-config/tree/master/css) in order to get this project approved.
  > I can see that you had added a stickler file but your PR is not running through the checks, I recommend you guys to check the [stickler](https://stickler-ci.com/) page to see if you had activated your project in there.

###### Your PR should look like this at the bottom: 
Screenshot from 2020-04-01 09-19-32

## [_Optional_] 

- [ ] set up a max-width or a fixed width so you element images would never brake no matter what.
##### On Zoom Out:
Screenshot from 2020-04-01 09-22-22

## Kindly address all these changes and submit a new code review

### Happy coding:computer: 

#### Daniel Laloush

**Comment by hillarioh** [comment]
The css fails is because of auto save from prettier extension, If you have a solution you can help me out

File: index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    </section>
+
+    <footer>
+      <div class="footer1">
+        <p>
+          Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an
+          <a href="#">Apple Retail Store</a>, or find a
+          <a href="#">reseller</a>.
+        </p>
+        <ul>
```
Comment (men32z) [inline index.html:78]: add a class with margin: 0  to this ul it's causing a white space in the top of the page, was really hard to find it xD

**Review by men32z — CHANGES_REQUESTED** [review]
Hey team!
## Good job on this project! 👍

There are still some changes 👇 you need to implement to approve this request 🔍

- [ ] fix the white space problem, I let you a comment of how to do it
- [ ] fix css problems, you can use  npx stylelint . --fix before to submit, if you have installed the CLI but if the problem is a prettier in your editor i recommend you to configure it,  remove it or find another. because is required that stickler checks pass.

## Please, make all REQUIRED changes and submit your request again! 😀

If you have any questions, feel free to send me a message through Slack (@men32z)
I would be happy to help! 👩‍💻

Happy coding 🤓

--Luis Preza

**Review by men32z — APPROVED** [review]
## Good job making the changes, you can merge.



### Status Approved

**Pull request merged**